resources/lang/de_DE/default.po

```diff
@@ -3150,3 +3150,34 @@ msgstr "Möchtest du den Arbeitseinsatz von %s wirklich löschen?"
 
 msgid "Number of shifts: %s"
 msgstr "Anzahl an Schichten: %s"
+
```

Comment:
I would highly appreciate it if you could move those above the translations with dot notation to be less confusing (and also move the `Number of shifts: %s` up too when you are on it ;))

Reply:
done